cfgov/cfgov/urls.py

```diff
@@ -289,6 +300,16 @@
     url(r'^token-provider/', token_provider),
 ]
 
+with flagged_urls('CCDB5_RELEASE') as _url:
+    apiBase = '^data-research/consumer-complaints/api/v1'
+    ccdb5_patterns = [
+        _url(apiBase + '/_suggest', CCDB5.suggest),
+        _url(apiBase + '/(?P<id>[0-9]+)$', CCDB5.document),
+        _url(apiBase, CCDB5.search)
+    ]
+urlpatterns += ccdb5_patterns
```

Comment:
This is the new format @willbarton communicated to me for loading a block of conditionally-flagged urls.  I don't think the `with` block works inside of an array definition